### PR TITLE
[NativeAOT-LLVM] Automate `WasmDebugging.csproj` via LLDB scripting

### PR DESCRIPTION
**Pull request by SingleAccretion**
This small (tiny) test has already found 1 bug in our debug info generation, and 2.5 bugs in Wasmtime debug info translation.

### CODE REVIEW

**Comment by SingleAccretion** [comment]
"Journey before destination".

@dotnet/nativeaot-llvm

File: src/tests/nativeaot/SmokeTests/HelloWasm/WasmDebugging.csproj [hunk 1]
```diff
@@ -2,12 +2,36 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <CLRTestKind>BuildAndRun</CLRTestKind>
-    <CLRTestPriority>0</CLRTestPriority>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <InvariantGlobalization>true</InvariantGlobalization>
     <RequiresProcessIsolation>true</RequiresProcessIsolation>
     <ReferenceXUnitWrapperGenerator>false</ReferenceXUnitWrapperGenerator>
+
+    <!-- Only works without optimizations now. -->
```
Comment (yowl) [inline src/tests/nativeaot/SmokeTests/HelloWasm/WasmDebugging.csproj]: Nit, reads as though it previously used to work with optimisations, so:
```suggestion
    <!-- Only works without optimizations for now. -->
```

File: src/tests/nativeaot/SmokeTests/HelloWasm/WasmDebuggingTestLauncher.cmd [hunk 2]
```diff
@@ -0,0 +1,7 @@
+echo ====== WasmDebuggingTestLauncher.cmd ======
```
Comment (yowl) [inline src/tests/nativeaot/SmokeTests/HelloWasm/WasmDebuggingTestLauncher.cmd:1]: As I understand it, we are not running the automatic testing in CI for windows due to a lack of `lldb`, so is this for manual running?

**Review by yowl — APPROVED** [review]
Good job, thanks!  Just one nit really.

File: src/tests/nativeaot/SmokeTests/HelloWasm/WasmDebuggingTestLauncher.cmd (hunk 2 above)
Comment (SingleAccretion) [reply src/tests/nativeaot/SmokeTests/HelloWasm/WasmDebuggingTestLauncher.cmd:1]: Yep.

**Pull request merged**